app-interface/manifests/rhose-flee-manager.yml

```diff
@@ -0,0 +1,126 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: rhose-fleet-manager
+  labels:
+    app: rhose-fleet-manager
+objects:
+  - apiVersion: apps/v1
+    kind: Deployment
+    metadata:
+      name: rhose-fleet-manager
+      labels:
+        app-component: rhose-fleet-manager
+    spec:
+      selector:
+        matchLabels:
+          app-component: rhose-fleet-manager
+      replicas: ${{FLEET_MANAGER_DEPLOYMENT_REPLICAS}}
+      template:
+        metadata:
+          labels:
+            app-component: rhose-fleet-manager
+        spec:
+          containers:
+            - image: ${{FLEET_MANAGER_IMAGE}}:${{{FLEET_MANAGER_IMAGE_TAG}}}
+              imagePullPolicy: IfNotPresent
+              name: rhose-fleet-manager
+              resources:
+                limits:
+                  cpu: ${{FLEET_MANAGER_CPU_LIMITS}}
+                  memory: ${{FLEET_MANAGER_MEM_LIMITS}}
+                requests:
+                  cpu: ${{FLEET_MANAGER_CPU_REQS}}
+                  memory: ${{FLEET_MANAGER_MEM_REQS}}
+              ports:
+                - containerPort: 8080
+              envFrom:
+                - configMapRef:
+                    name: fleet-manager-bridge-config
+                - secretRef:
+                    name: fleet-manager-bridge-secrets
+              livenessProbe:
+                httpGet:
+                  path: /q/health/live
+                  port: 8080
+                initialDelaySeconds: 10
+                periodSeconds: 10
+              readinessProbe:
+                httpGet:
+                  path: /q/health/ready
+                  port: 8080
+                initialDelaySeconds: 10
+                periodSeconds: 10
+
+  - apiVersion: v1
+    kind: Service
+    metadata:
+      name: fleet-manager-bridge
+    spec:
+      ports:
+        - name: web
+          protocol: TCP
+          port: 8080
+          targetPort: 8080
+      selector:
+        app-component: fleet-manager
+
+  - apiVersion: v1
+    kind: ConfigMap
+    metadata:
+      name: fleet-manager-bridge-config
+    data:
+      EVENT_BRIDGE_DB_USERNAME: ${{EVENT_BRIDGE_DB_USERNAME}}
+      EVENT_BRIDGE_DB_PASSWORD: ${{EVENT_BRIDGE_DB_PASSWORD}}
+      EVENT_BRIDGE_DB_HOST: ${{EVENT_BRIDGE_DB_HOST}}
+      EVENT_BRIDGE_DB_PORT: ${{EVENT_BRIDGE_DB_PORT}}
+      EVENT_BRIDGE_DB_SCHEMA: ${{EVENT_BRIDGE_DB_SCHEMA}}
+      EVENT_BRIDGE_SSO_URL: ${{EVENT_BRIDGE_SSO_URL}}
+      EVENT_BRIDGE_SSO_CLIENT_ID: ${{EVENT_BRIDGE_SSO_CLIENT_ID}}
+
+parameters:
+  - name: FLEET_MANAGER_IMAGE
+    value: "quay.io/app-sre/rho-fleet-manager"
```

Comment:
`rhose-fleet-manager` as the image name? I can't remember if we have decided this already, sorry.

Reply:
@robpblake  Not really yet but that can be changed later. I'm just reflecting what we have here: https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/app-interface/app_sre_fleet_manager_build_push.sh#L6

Reply:
OK, thanks for the reminder!